packages/gitlab/data_stream/api/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,217 @@
+---
+description: Pipeline to process gitlab api logs
+processors:
+- set:
+    field: ecs.version
+    tag: set_ecs_version
+    value: 8.11.0
+- rename:
+    field: message
+    target_field: event.original
+    tag: rename_message
+    ignore_missing: true
+    if: ctx.event?.original == null
+- remove:
+    field: message
+    ignore_missing: true
+    tag: remove_message
+    if: ctx.event?.original != null
+- drop: 
+    if: "ctx.event.original.startsWith('#')"
+    description: "Drop if logline contains header(s), which startswith `#`"
+- json:
+    field: event.original
+    tag: json_original
+    target_field: gitlab.api
+- date:
+    field: gitlab.api.time
+    target_field: '@timestamp'
+    formats:
+    - ISO8601
+    ignore_failure: true
+- rename:
+    field: gitlab.api.severity
+    target_field: log.level
+    ignore_missing: true
+- script:
+    lang: painless
+    source: |-
+      if (ctx.gitlab?.api?.duration_s != null) {
+        ctx.event.duration = ctx.gitlab.api.duration_s * 1000000;
+      }
+    ignore_failure: true
+- rename:
+    field: gitlab.api.status
+    target_field: http.response.status_code
+    ignore_missing: true
+- rename:
+    field: gitlab.api.method
+    target_field: http.request.method
+    ignore_missing: true
+- rename:
+    field: gitlab.api.path
+    target_field: url.path
+    ignore_missing: true
+- rename:
+    field: gitlab.api.host
+    target_field: url.domain
+    ignore_missing: true
+- dot_expander:
+    path: gitlab.api
+    field: meta.caller_id
+- dot_expander:
+    path: gitlab.api
+    field: meta.client_id
+- dot_expander:
+    path: gitlab.api
+    field: meta.feature_category
+- dot_expander:
+    path: gitlab.api
+    field: meta.remote_ip
+- convert:
+    field: gitlab.api.meta.remote_ip
+    type: ip
+    ignore_missing: true
+    on_failure:
+        - remove:
+            field: gitlab.api.meta.remote_ip
+            ignore_missing: true
+        - append:
+            field: error.message
+            value: '{{{_ingest.on_failure_message}}}'
+- dot_expander:
+    path: gitlab.api
+    field: meta.user
+- dot_expander:
+    path: gitlab.api
+    field: meta.user_id
+- rename:
+    field: gitlab.api.meta.caller_id
+    target_field: event.provider
+    ignore_missing: true
+- split:
+    field: gitlab.api.remote_ip
+    if: ctx.gitlab?.api?.remote_ip != null && ctx.gitlab?.api?.remote_ip != ''
+    tag: 'split_ips'
+    target_field: source.ip
+    separator: ', '
```

Are we confident that there will always be a space?

Maybe
```suggestion
    separator: ', ?'
```
?

---

There was in all the samples I have, but I guess better safe than sorry?